avocado_virt/plugins/virt_bootstrap.py

```diff
@@ -64,6 +64,7 @@ def run(self, args):
         except Exception, exc:
             LOG.error('Failed to get SHA1 from file: %s', exc)
             fail = True
+            sha1 = "FAILED TO GET DOWNLOADED FROM AVOCAOD-PROJECT"
```

Comment:
There's a typo here.  Also, I bet we won't get collisions here but, but consider using a flag instead, such as `sha1 = None`.

Reply:
I'd prefer the readable string rather than special values which require comments and later in code we'd have to remember that  `sha1 = None` means this, `actual_sha1 = None` means not found locally...